internal/api/ui/login/custom_action.go

```diff
@@ -23,49 +23,60 @@ func (l *Login) runPostExternalAuthenticationActions(
 	httpRequest *http.Request,
 	idpUser idp.User,
 	authenticationError error,
-) (*domain.ExternalUser, error) {
+) (*domain.ExternalUser, bool, error) {
 	ctx := httpRequest.Context()
+	change := false
```

Comment:
i think we should name the return values and (as you already mentioned) the externalUser as return value is not necessary, but does not have to be removed now
for the `change` i'd suggest to rename it to `userChanged` or something like that

Reply:
renamed to userChanged and used it as named return value, also corrected the handling in the ldap provider